source/technical/swift.rst

```diff
@@ -201,3 +201,34 @@ There is no CLI command for creating a Folder within a Container. However, you m
 .. code-block:: bash
 
    openstack object create --name "MyFolder/MyObject.txt" MyContainer MyObject.txt
+
+__________________________________
+Mounting Object Store
+__________________________________
+
+When logging into an instance using Chameleon Cloud supported images, such as ``CC-CentOS7`` and ``CC-Ubuntu16.04``, you may use the pre-installed ``cloudfuse`` (Source: `Github <https://github.com/redbo/cloudfuse>`_) to mount your Chameleon Object Store as directory on you Linux environment.
+
+Before mount, create ``~/.cloudfuse`` file with the following content:
+
+.. code-block:: bash
+
+   username=<username>
+   password=<password>
+   tenant=Chameleon
```

Comment:
This should be the project name of the user, not Chameleon which is a staff-only project.